transaction-volume-anomaly-detection/src/agent.js

```diff
@@ -0,0 +1,355 @@
+const {
+  Finding,
+  FindingSeverity,
+  FindingType,
+  getEthersProvider,
+  getTransactionReceipt,
+} = require("forta-agent");
+
+const {
+  getContractsByChainId,
+  globalSensitivity,
+  bucketBlockSize,
+} = require("./agent.config");
+
+const ARIMA_SETTINGS = {
+  p: 2,
+  d: 1,
+  q: 2,
+  P: 1,
+  D: 0,
+  Q: 1,
+  s: 50,
+  verbose: false,
+};
+const ARIMA = require("arima");
+const provider = getEthersProvider();
+const contractTracker = {};
+const timestampThreshold = bucketBlockSize * 60 * 60 * 60; // The timestamp time in seconds based on the bucket block size and blockTime
```

Comment:
My idea was something like
1. store the blocktime for all chains (not sure how to do it on L2s)
2. get the bucketBlockSize from the config
3. calculate the time for one entry (bucketBlockSize * blocktime).
And then we should set the seasons based on that time